tinygrad/codegen/linearizer.py

```diff
@@ -215,7 +214,7 @@ def calc_tc_idxs(local_sizes: List[int], aliases: List[List[int]]):
       alias_buf_idxs.append((i, localbuf_idx, buf_idxs,))
 
     # reduce loop
-    loop_ctx = self.render_loop(reduce_idxs, 2)
+    loop_ctx = self.render_loop(reduce_idxs, seq*4+2)
```

Comment:
what's the reasoning being doing seq*4

Reply:
@chaosagent for parallel reduces

It looked like `depth` was a way to fuse loops from multiple reduceops; we didn't come to a conclusion but it seemed like the thinking around parallel/sequential reduceops is that `self.reduceops` would become a 2D list. so a `seq` number would be shared amongst parallel and incremented when we went to the next group of parallel reduceops

`*4` because it looked like there were more render loops but this could probably be `*2`